compiler/luci/service/src/Nodes/CircleReshape.cpp

```diff
@@ -120,8 +120,7 @@ loco::TensorShape Algorithm::visit(const luci::CircleReshape *node)
       // for non-existing `shape`, we can use `newShape` if it's valid
       auto new_shape = node->newShape();
       auto rank = new_shape->rank();
-      auto shape_dummy = dynamic_cast<luci::CircleOutputDummy *>(node->shape());
-      if (shape_dummy && rank > 0)
+      if (rank > 0)
```

Comment:
Q) why did you remove to check `shape_dummy` ?

Reply:
For my model converted from Pytorch I would like to [use dimensions stored in newShape](https://github.com/Samsung/ONE/blob/e5e151d547616f1ad3be6ff13f86b53990b7c157/compiler/luci/service/src/Nodes/CircleReshape.cpp#L126-L137) but models converted from Pytorch have no `luci::CircleOutputDummy`

Reply:
Do you know what and how `CircleOutputDummy ` works?

Reply:
I know only (based on the spec) that it's "Temporary DummyNode used with dangle CircleNode" but to be honest I don't know why it is used here ;)

Reply:
If you don't know the purpose of this node, why did you deleted it?

Reply:
Just for clarification - I believe I understand concept of `CircleOutputDummy` here -> we are using values from attribute only if shape input to missing (dummy node assign to avoid dangling operand) -> attribute here is to reflect TF impl of Reshape node.

**I am just not understand why we cannot relax the algorithm** -> if based on shape input we can only calculate rank, let's check if maybe we can extract sth valuable from attribute (still treating input data with the higher priority)